### DISCUSSION TITLE
Anyone Want to Help With Book Reviews?

### DISCUSSION THREAD

**Discussion by robmoffat** [risk-first/website#125] [Announcements]
Hi Everyone,

I am currently working through extensive edits to [Risk First Software Development, Second Edition](https://github.com/risk-first/website/discussions/113), which promises to be a big upgrade from [the first book](https://www.amazon.com/Risk-First-Software-Development-Rob-Moffat-ebook/dp/B07MK9LTHN/ref=sr_1_1?crid=1W3R85DQOLYU5&dib=eyJ2IjoiMSJ9.IoKviuuHbG66QOZQOugJE89giLvRqhoKztVC4OpZ6HOFBbjbIhOm54YASpoBB8DoMN9nATIBilcv6OoVc8yELjHzOEoEfXxLm2qZnF7ErQXJMXrRtRH36NO_8ZtBM-mnfwvys3nJVUXem6bYxtO4X2j9TdVwTec389_ZkB46elBCwIawTr7duPsxn_QXMsbBM4OzloAUIN9cilc9WnO-WZZotB_rGFqgfK9F2XlO4aM.rPyhNvtgtVlXalUU4bMaQn_JHYxr0Pkpse0Ggh4RV9Y&dib_tag=se&keywords=risk+first+software+development&qid=1728580414&sprefix=risk+first+software+development%2Caps%2C138&sr=8-1) and also quite a change in content from [The Website](https://riskfirst.org).  

My editor team over at [The Pragmatic Bookshelf](https://pragprog.com) have asked me to assemble a varied selection of reviewers for the first six chapters - a mid-point review of sorts.  I'm leaning on my existing network of Finance / Open Source colleagues for this but since it's all about getting as many perspectives as possible, I wanted to open this up to anyone who's joined the Risk-First team.

Obviously, if you help me out with this I will credit your work in the book and on the Risk-First website too: should be just some reading and passing back your comments (I expect anyway).

If you fancy helping out then drop a comment here and I'll get in touch.  

thanks!

Rob

**Comment by jbampton** [comment]
Hi @robmoffat I would like to read the first six chapters and hope to provide some feedback.

Well done on the new book !!

**Comment by DianeHolmes** [comment]
Dear Rob,

I’m really excited you’re writing a second edition. Yay!

 Looking at software as risk control was a game changer for me.

I’m happy to help with the review.



Best regards,
Akasia

**Comment by allanwsilva** [comment]
I'm happy to help too.

**Comment by robmoffat** [comment]
Thanks so much @allanwsilva @DianeHolmes  and @jbampton !

I thought there was a way to message direct in GitHub but it appears not, so could you just ping me an email to [drench_medical.0q@icloud.com](mailto:drench_medical.0q@icloud.com) and I can put you in touch with the editor to start the process off.

Looking forward to hearing your feedback!!

Rob
